### PR TITLE
fix: arrange unschedule session based on their event-time

### LINKED ISSUE #3702: My Sessions: Shows past sessions in unscheduled session of past event in tab "Upcoming"
My Sessions: Shows past sessions in unscheduled session of past event in tab "Upcoming"
The private link is at https://eventyay.com/my-sessions/upcoming.
Furthermore the sessions shows up 4 x times.

EXPECTED: Even though the session is unscheduled, the system should take the time of the overall event. If the event is past the unscheduled session should also show up in the "past tab".

Screenshot from 2019-12-15 18-30-13

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3702 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/my-sessions/list.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default Route.extend(AuthenticatedRouteMixin, {
     this.set('params', params);
     let filterOptions = [];
     if (params.session_status === 'upcoming') {
-      filterOptions = [
+      filterOptions.push(
```
Comment (kushthedude) [inline app/routes/my-sessions/list.js]: Dont refactor them. Just add the filter parameters

Comment (maze-runnar) [reply app/routes/my-sessions/list.js]: Is this wrong?

**Comment by kushthedude** [comment]
No its not wrong but keep one issue at a time, changing the array insertion
to push may also get shown in file blame which is not related to the PR.

**Comment by kushthedude** [comment]
An after/before gif ?

**Comment by maze-runnar** [comment]
> An after/before gif ?

https://www.youtube.com/watch?v=E0R4FfHvIjs

**Comment by kushthedude** [comment]
Looks good to me, squash the commits.

**Comment by maze-runnar** [comment]
> Looks good to me, squash the commits.

Done! please merge .

**Pull request merged**